.github/workflows/prs-user-limit.yml

```diff
@@ -0,0 +1,18 @@
+name: "[service] Check open pull-requests per user"
+
+on:
+  pull_request_target:
+    types: [opened]
+
+jobs:
+  comment:
+    if: github.repository == 'conan-io/conan-center-index'
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: master
+      - uses: ./.github/actions/check-open-prs-per-user
+        with:
+          threshold: "5"
```

Comment:
@czoido What threshold do you think we should use? I would say 5 is too low, and maybe 10 is better, but not Dani nor me know what'd be best, happy to hear your opinion

Reply:
Maybe 8? That's in the middle...

Reply:
```suggestion
          threshold: "8"
```